### PR TITLE
Administration files

### PR DESCRIPTION
**Pull request by Sylv11**

### CODE REVIEW

**Comment by jasonvanmalder** [comment]
[FE: Système de fichier](https://trello.com/c/ZzyV0i1l/165-fe-syst%C3%A8me-de-fichier)

File: src/components/administration/common/AdministrationLog.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const Level = styled.span`
     border-radius: 50%;
 `
 
-const AdministrationLog = (props) => {
+const AdministrationLog = props => {
```
Comment (jasonvanmalder) [inline src/components/administration/common/AdministrationLog.js]:
```suggestion
const AdministrationLog = ({logs}) => {
```

File: src/components/administration/common/AdministrationLog.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const Level = styled.span`
...
 
     const {log} = props
```
Comment (jasonvanmalder) [inline src/components/administration/common/AdministrationLog.js]:
```suggestion
```

**Comment by jasonvanmalder** [comment]
Rename FileComponentViewer into HighlightedFiles

File: src/components/administration/common/AdministrationLog.js (hunk 1 above)
Comment (Sylv11) [reply src/components/administration/common/AdministrationLog.js]: But further in the code, there is a props.history.push so we need props and not only {logs}

Or I suggest : 
const AdministrationLog = ({log, history}) => {

}

Comment (Sylv11) [reply src/components/administration/common/AdministrationLog.js]: And therefore use `history.push` instead of `props.history.push`

Comment (jasonvanmalder) [reply src/components/administration/common/AdministrationLog.js]: ({logs, history: {push}}) ;)

Comment (Sylv11) [reply src/components/administration/common/AdministrationLog.js]: Ow it looks sexy like me

Ok I change it for the next commit

`push(/fuk_u_jackson)`

Comment (jasonvanmalder) [reply src/components/administration/common/AdministrationLog.js]: You dead

**Comment by jasonvanmalder** [comment]
[FE: Lister les fichiers (liste, icônes)](https://trello.com/c/OmHtJrVW/21-fe-lister-les-fichiers-liste-ic%C3%B4nes)

**Comment by jasonvanmalder** [comment]
Fix conflict (same for back-end)

**Pull request merged**